keps/NNNN-kep-template/README.md

```diff
@@ -560,9 +560,13 @@ previous answers based on experience in the field.
 ###### How can an operator determine if the feature is in use by workloads?
 
 <!--
-Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-checking if there are objects with field X set) may be a last resort. Avoid
-logs or events for this purpose.
+If you are using a feature-gate, then you do not have to answer this question, 
```

Comment:
"In use by workloads" != "enabled", so this isn't really correct. This is more getting at - oh, there is a metric in kube-state-metrics showing that X resource exists, etc.